apps/nextjs/src/components/AppComponents/Chat/markdown.tsx

```diff
@@ -18,15 +18,17 @@ const MemoizedReactMarkdown: FC<Options> = memo(
     prevProps.className === nextProps.className,
 );
 
+export type ReactMarkdownWithStylesProps = Readonly<{
+  markdown: string;
+  lessonPlanSectionDescription?: string;
+  className?: string;
+}>;
```

Comment:
It's nice to extract this to a named type outside the function. Make sense

Reply:
If we like this pattern we might want to do a pass on all components to follow the same approach.

Reply:
I'm personally a fan of it! As part of my "you should be able to scan what a function does without too much scrolling" guideline